src/css/custom.scss

```diff
@@ -105,6 +105,45 @@ h4 {
   }
 }
 
+html, body {
+  min-height: 100%;
+  margin: 0;
+  padding: 0;
+  background-image: linear-gradient(50deg, #0E2515, #25392b);
+  color: #E0E0E0;
+  background-attachment: fixed;
+  width: 100%;
+  overflow-x: hidden;
+}
+
+/* Adjusting the navbar to blend with the background */
+/* Adjusting the navbar to blend with the new background gradient */
+/* Adjusting the navbar to blend with the background */
+.navbar {
+  background-image: linear-gradient(50deg, #0E2515, #25392b);
+  backdrop-filter: blur(5px); /* If you want a blur effect */
+  z-index: 100;
+  /* Other styles */
```

Comment:
Can we remove these comments from the css file?

Reply:
ok